docs/site/tutorials/core/10-advanced-recipes.md

```diff
@@ -0,0 +1,136 @@
+---
+lang: en
+title: 'Advanced Recipes'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/core-tutorial-part10.html
+---
+
+There are a few advanced usages of the LoopBack core modules that are not
+covered by the Greeter Extension and Greeter Application example.
+
+# Create your own decorator
+
+You can create your own decorator:
+
+1. Create a new decorator from scratch by using `DecoratorFactory` from
+   `@loopback/metadata`. See example from
+   [custom-inject-decorator.ts](https://github.com/strongloop/loopback-next/blob/master/examples/context/src/custom-inject-decorator.ts)
+
+2. Create a sugar decorator for an existing decorator.
+
+   ```ts
+   export function globalInterceptor(group?: string) {
+     return bind({tags: {}});
+   }
+   ```
+
+# Create your own injector
+
+```ts
```

Comment:
can we add a link to the example code here? Or makes it clear where do we put the code?

Reply:
@jannyHou, i'm trying to find example code in https://github.com/strongloop/loopback-next/tree/master/examples/context but couldn't find any.  Do you know where we can find the example code? Thanks.

Reply:
@dhmlau I think this one https://github.com/strongloop/loopback-next/blob/master/examples/context/src/custom-inject-decorator.ts is a custom injector.